packages/backend/migrations/20250114010822_test_first_five_view.down.sql

```diff
@@ -0,0 +1,12 @@
+-- Add down migration script here
+SELECT *
+FROM users
+LIMIT 2;
+
+SELECT * 
+FROM permission_level
+LIMIT 2; 
+
+SELECT * 
+FROM permissions
+LIMIT 2; 
```

Comment:
Why is this in a down migration script? It looks like a test of some kind, but that doesn't make sense to put in a migration.